### PR TITLE
doc/mgr: Clean up orchestrator.rst (spell/punct, awkward wording)

### PR DESCRIPTION
**Pull request by dmick**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>
jenkins do not test
jenkins render docs

### CODE REVIEW

**Comment by dmick** [comment]
jenkins do not test

**Comment by dmick** [comment]
jenkins render docs

**Comment by dmick** [comment]
jenkins do not test

**Comment by dmick** [comment]
jenkins render docs

**Comment by dmick** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
jenkins test make check

File: doc/mgr/orchestrator.rst [hunk 1]
```diff
@@ -39,15 +39,14 @@ for the orchestrator module is needed.
...
+* A *service* has a specific *service type*
 * A *daemon* is a physical instance of a *service type*
 
 
 .. note::
 
     Orchestrator modules may only implement a subset of the commands listed below.
-    Also, the implementation of the commands are orchestrator module dependent and will
-    differ between implementations.
+    Also, the implementation of the commands may differ differ between modules.
```
Comment (varshar16) [inline doc/mgr/orchestrator.rst:49]:
```suggestion
    Also, the implementation of the commands may differ between modules.
```

Comment (dmick) [reply doc/mgr/orchestrator.rst:49]: thanks!

Comment (dmick) [reply doc/mgr/orchestrator.rst]: forcepushed the update

**Review by varshar16 — CHANGES_REQUESTED** [review]
@dmick please rebase the commit

**Comment by dmick** [comment]
rebased, reworded conflicting change text to avoid awkward phrasing

**Review by varshar16 — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
Wasn't on my radar, due to missing github labels.

**Pull request merged**